test-runner-frontend/package-lock.json

```diff
@@ -3997,9 +3997,9 @@
       }
     },
     "@types/react-dom": {
-      "version": "18.0.5",
-      "resolved": "https://artifactory.twitter.biz/artifactory/api/npm/js-virtual/@types/react-dom/-/react-dom-18.0.5.tgz",
-      "integrity": "sha512-OWPWTUrY/NIrjsAPkAk1wW9LZeIjSvkXRhclsFO8CZcZGCOg2G0YZy4ft+rOyYxy8B7ui5iZzi9OkDebZ7/QSA==",
+      "version": "18.0.6",
+      "resolved": "https://artifactory.twitter.biz/artifactory/api/npm/js-virtual/@types/react-dom/-/react-dom-18.0.6.tgz",
+      "integrity": "sha512-/5OFZgfIPSwy+YuIBP/FgJnQnsxhZhjjrnxudMddeblOouIodEQ75X14Rr4wGSG/bknL+Omy9iWlLo1u/9GzAA==",
       "dev": true,
       "requires": {
         "@types/react": "*"
```

Comment:
can we please remove the changes from this file? you only usually want to commit package-lock.json files when you update package.json

Reply:
Done